instrumentation/opentelemetry-instrumentation-aiohttp-client/src/opentelemetry/instrumentation/aiohttp_client/__init__.py

```diff
@@ -230,8 +229,8 @@ async def on_request_start(
             trace_config_ctx.span = None
             return
 
-        http_method = params.method
-        request_span_name = _get_span_name(http_method)
+        method = params.method
+        request_span_name = _get_span_name(method)
```

Comment:
Probably don't need this anymore.

Reply:
`request_span_name` is used as span name to start span